llvm/test/CodeGen/AMDGPU/waitcnt-debug-non-first-terminators.mir

```diff
@@ -0,0 +1,21 @@
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -run-pass si-insert-waitcnts -amdgpu-waitcnt-forcezero=1 %s -o - | FileCheck %s
```

Comment:
Just use update_mir_test_checks? The test doesn't show me where the waitcnt is

Reply:
For some reason the script generates nothing for this test case. Not very sure what's going on.

Reply:
The whole point of this PR is to make sure there is no waitcnt between any terminator.

Reply:
Aha! That does it.